### PR TITLE
News authorization and authentication

### PR DESCRIPTION
**Pull request by nbalicka35**
Now requests to server are checking user privileges and whether user token is valid

### CODE REVIEW

File: backend/src/main/java/backend/lezaczek/Repositories/UsersRepository.java
Comment (karomnii) [inline backend/src/main/java/backend/lezaczek/Repositories/UsersRepository.java:1]: Is this query needed? there is a built in function that works like this

File: backend/src/main/java/backend/lezaczek/Services/UserService.java
Comment (karomnii) [inline backend/src/main/java/backend/lezaczek/Services/UserService.java:1]: That is not needed too

Comment (nbalicka35) [reply backend/src/main/java/backend/lezaczek/Services/UserService.java:1]: what is unnecessary and why? "that" doesn't say anything

File: backend/src/main/java/backend/lezaczek/Repositories/UsersRepository.java
Comment (nbalicka35) [reply backend/src/main/java/backend/lezaczek/Repositories/UsersRepository.java:1]: working on that

**Review by karomnii — APPROVED** [review]

**Pull request merged**